Marlin/ultralcd_impl_DOGM.h

```diff
@@ -587,7 +575,8 @@ static void lcd_implementation_status_screen() {
     uint8_t vallen = lcd_strlen(value);
 
     #if ENABLED(USE_BIG_EDIT_FONT)
-      if (lcd_strlen_P(pstr) <= LCD_WIDTH_EDIT - 1) {
+      //if (lcd_strlen_P(pstr) <= LCD_WIDTH_EDIT - 1) {
```

Comment:
If replacing a line, you can just remove it instead of leaving it commented out.